bspump/kafka/source.py

```diff
@@ -151,17 +155,15 @@ async def main(self):
 						self.LastRefreshTopicsTime = current_time
```

Comment:
What is the reason? ChatGPT?😀
This has to be strongly tested before it is merged…

Reply:
Which line you are commenting? :D

Reply:
Only the green or red ones are changed.